src/Microsoft.Graph.Core/Models/AsyncOperationStatus.cs

```diff
@@ -2,42 +2,38 @@
 //  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 // ------------------------------------------------------------------------------
```

Comment:
@shiftylogic When I cloned this to my machine, everything seemed to be in order. This seems like it's fine to merge, not sure what @ginach is referencing.